libs/components/datetime/src/lib/modules/datepicker/datepicker-hint-token.ts

```diff
@@ -0,0 +1,9 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * @internal
+ */
+export const SKY_DATEPICKER_HINT_CONFIG = new InjectionToken<boolean>(
```

Comment:
I don't know that I love the term `SKY_DATEPICKER_HINT_CONFIG`. This is just a boolean that is hiding the hint text for datepicker and not really a "config".

Taking inspiration from `SKY_FORM_ERRORS_ENABLED` - what if we called this `SKY_DATEPICKER_HINT_TEXT_HIDDEN`?

Reply:
Updated